src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -17,3 +22,58 @@ const srSubstitute = (srIgnored, substitutionText) => {
 };
 
 export default srSubstitute;
+
+export const hasGuardOrReservePeriod = (formData) => {
+  const serviceHistory = formData.servicePeriods;
```

Comment:
I think this needs to actually look at `formData.serviceInformation.servicePeriods`

Reply:
Good catch. For now, I changed the form config page to pass the service information object, rather than create a separate helper.